### ISSUE TITLE
AOR options in relation to field types

### ISSUE THREAD

**Issue by willrennie** [salesagility/SuiteCRM#439]
Options should be restricted to field types. For example, I can perform a sum on the sales stage, which outputs as 0 as this is a drop down with values such as 'Closed Won'. This shouldn't really be an option, although does not affect the report outcome or cause any related issues.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [AOR options in relation to field types](https://trello.com/c/EDX8ZBHu/149-aor-options-in-relation-to-field-types)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
